gui/formatload.cc

```diff
@@ -67,13 +67,11 @@ bool FormatLoad::processFormat(Format& format)
   if (hfields.size() < 5) {
     return false;
   }
-  QString htmlPage = lines_[currentLine_++];
-  htmlPage.replace(QRegExp("^[\\s]*"), "");
-  htmlPage.replace(QRegExp("[\\s]$"), "");
+  QString htmlPage = lines_[currentLine_++].trimmed();
 
-  QRegExp regex("^option");
+  QRegularExpression regex("^option");
   QList <FormatOption> optionList;
-  while (currentLine_ <lines_.size() && regex.indexIn(lines_[currentLine_]) == 0) {
+  while ((currentLine_ < lines_.size()) && regex.match(lines_[currentLine_]).hasMatch()) {
```

Comment:
So many of these regex uses are overkill. Isn't this currentline_.startswith("option") ? There are a few occurrences like this.

Reply:
yes.